README.md

```diff
@@ -24,21 +24,38 @@ To run that demo on your own computer:
 * `npm run storybook`
 * Visit http://localhost:9001/
 
-## How to Use
+## Getting Started
+#### Install dependencies
+```
+npm install --save react-dates
+```
+Make sure you have the following dependencies installed:
+```
+ npm install --save moment react react-dom react-addons-shallow-compare
```

Comment:
would it be possible to abstract this into an package.json script? some of us are on windows systems so ideally the command would be compatible with windows, as well? Also, I could not get this command to play nicely with yarn: `yarn info react-dates peerDependencies --json | command sed 's/[\{\},]//g ; s/: /@/g; s/ *//g' | xargs yarn add react-dates`

Here is the output for me from running just the following `yarn info react-dates peerDependencies --json | command sed 's/[\{\},]//g ; s/: /@/g; s/ *//g'`:

`"type":"inspect""data":"moment":"2.10-2.14||^2.15.1""react":">=0.14""react-dom":">=0.14""react-addons-shallow-compare":">=0.14"
"type":"finished""data":630`